Server/main.cpp

```diff
@@ -68,10 +52,10 @@ int main(int argc, char **argv) {
 			std::cout << "sent object " << gameObject->getId() << " to " << c->getId() << std::endl;
 		}
 
-		gameEngine.addGameObject(new Player(vec3(0.0f), vec3(0.0f), vec3(0.0f,0.0f,-1.0f), c->getId(), 1));
-
-		// Allow the newly connected player to move the ball.
-		c->on(NetMessage::BALL_X, handleBallMove);
+		auto player = new Player(origin, origin, c->getId(), 1.0f);
+		player->setDirection(vec3(0, 0, -1));
```

Comment:
Doesn't the player already have its direction set in line 6 of the `Shared/Player.cpp` file?

Reply:
That's for a different constructor. We can change this to use that constructor. But, I feel like this is more explicit instead of having a list of random values in the constructor.